lms/static/sass/elements/_system-feedback.scss

```diff
@@ -4,7 +4,7 @@
 // pre-pattern library messages
 
 // UI : message
-.wrapper-msg {
+%wrapper-msg {
```

Comment:
after reviewing with Leslie, the suggestion here would be to have margin-top: $baseline instead of margin: $baseline. 

The desired effect is to have the fixed message be full width, minus the 20px of margin on top currently being added.

Screenshot:

![image](https://cloud.githubusercontent.com/assets/2023680/9494778/fb6ffc9e-4bd5-11e5-9c16-d5c43f924e82.png)